kroxylicious/src/main/java/io/kroxylicious/proxy/filter/NetFilter.java

```diff
@@ -23,18 +23,23 @@ public interface NetFilter {
 
     interface NetFilterContext {
         /**
-         * @return The address of the client
-         * (possibly not the address of the remote TCP/TLS peer, if any intermediate L4 or
-         * L7 proxy is propagating client connection information).
+         * @return The source host of the client, taking into account source host information
+         * propagated by intermediate proxies.
+         * You can think of this as being like HTTP's {@code X-Forwarded-For} header.
```

Comment:
Since your quoted `@return` no longer matches the current doc, I pushed something while you were reviewing. Let me know if you think the current text is better.